src/main/java/com/iota/iri/service/tipselection/impl/TipSelectorImpl.java

```diff
@@ -11,11 +11,7 @@
 import com.iota.iri.model.Hash;
 import com.iota.iri.service.ledger.LedgerService;
 import com.iota.iri.service.snapshot.SnapshotProvider;
-import com.iota.iri.service.tipselection.EntryPointSelector;
-import com.iota.iri.service.tipselection.RatingCalculator;
-import com.iota.iri.service.tipselection.TipSelector;
-import com.iota.iri.service.tipselection.WalkValidator;
-import com.iota.iri.service.tipselection.Walker;
+import com.iota.iri.service.tipselection.*;
```

Comment:
Not really a review on this PR specifically, but can we do something about Codacy freaking out about these kinds of imports?

Reply:
yeah I know...
I need to find time to take care of this, so much to do...